Examples/LambdaFunctions/serverless/ErrorHandling-template.yml

```diff
@@ -0,0 +1,20 @@
+service: errorhandling-swift-aws
+
+package:
+    artifact: .build/lambda/ErrorHandling/lambda.zip
+
+provider:
+    name: aws
+    runtime: provided
+    iamRoleStatements:
+        - Effect: Allow
+          Action:
+              - logs:CreateLogGroup
+              - logs:CreateLogStream
+              - logs:PutLogEvents
+          Resource: "*"        
+
+functions:
+    errorHandlingFunction:
+        handler: ErrorHandling
+        memorySize: 128
```

Comment:
others templates do not specify men size. intentional?

Reply:
Good spot! `memorySize` is not mandatory, if omitted it will get the default one.
It's better to put one to be sure about the costs and the cpu associated with the deployment.

I'll fix it where is missing.

Note:
The `128 MB` memory size could not be perfect as each lambda must be fine tuned to optimise the performance/costs tradeoff.
- The memory is proportional to the cpu assigned to the lambda.
- The cost is evaluated with the duration and the memory associated to the lambda.
In practice it's required to test the lambda to give the right amount of memory without incurring in major costs.

https://docs.aws.amazon.com/lambda/latest/dg/configuration-console.html
https://aws.amazon.com/lambda/pricing/

Reply:
001b55a